modulemd/modulemd-module-stream.c

```diff
@@ -254,6 +254,7 @@ modulemd_module_stream_read_yaml (yaml_parser_t *parser,
                    MODULEMD_YAML_ERROR_PARSE,
                    "Unknown ModuleStream version: %" PRIu64,
                    modulemd_subdocument_info_get_mdversion (subdoc));
+      return NULL;
       break;
```

Comment:
Good catch!

But what are your thoughts on removing this `break` immediately following the `return`? I'm almost surprised the clang auto-formatting tools don't remove it or warn that it's unreachable code.

Reply:
I'd prefer to leave it in. A missing `break` can cause all sorts of havoc on a refactor. Better to have it and not need it than to need it and not have it.